dependency/test/test.sh

```diff
@@ -0,0 +1,83 @@
+
+
+#!/bin/bash
+
+set -euo pipefail
+
+extract_tarball() {
+  rm -rf icu
+  mkdir icu
+  tar --extract \
+    --file "$1" \
+    --directory icu
+}
+
+set_ld_library_path() {
+  export LD_LIBRARY_PATH="$PWD/icu/lib:${LD_LIBRARY_PATH:-}"
+}
+
+check_version() {
+  expected_version=$1
+  actual_version="$(/icu/bin/icuinfo 2> /dev/null | sed -rn 's/.*param name="version">([0-9\.]+).*/\1/p')"
+  if [[ "${actual_version}" != "${expected_version}" ]]; then
```

Comment:
seems like the tests error right here for me, and also they generate an ICU directory in `test` that doesn't get cleaned up